### PR TITLE
Docs: Getting started edits

### PR DESCRIPTION
**Pull request by oddlittlebird**
Adjusted weights, moved some from mac.md to configuration.md, have some questions

**What this PR does / why we need it**:
- Adjusts Installation guide weights to make future organization easier
- Moves some configuration file from the Mac installation guide to the Configuration topic
- Makes minor edits to content as I start trying to understand what this section needs

**Which issue(s) this PR fixes**:
Starts on epic #19893

**Special notes for your reviewer**:

Please note the code comment questions.

### CODE REVIEW

File: docs/sources/installation/configuration.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ specified in a `.ini` configuration file or specified using environment variable
 
 > **Note.** Grafana needs to be restarted for any configuration changes to take effect.
```
Comment (oddlittlebird) [inline docs/sources/installation/configuration.md]: Why is this topic here rather than in Getting Started?

File: docs/sources/installation/mac.md [hunk 2]
```diff
@@ -62,24 +64,7 @@ brew services start grafana
 
 Default login and password `admin`/ `admin`
 
-
```
Comment (oddlittlebird) [inline docs/sources/installation/mac.md]: All removed content was moved to configuration.md. If it is generic rather than Mac-specific, then we should remove the "Mac"

File: docs/sources/installation/requirements.md [hunk 3]
```diff
@@ -47,29 +47,25 @@ Grafana supports the following databases:
...
+> Note: Running Grafana without JavaScript enabled in the browser is not supported.
 
-> Note 2: Internet Explorer 11 is only fully supported in Grafana versions prior v6.0.
-
-> Note 3: Running Grafana without JavaScript enabled in the browser is not supported
-
-### Known issues
-
-#### Problem with logging in using Safari 12
+### Known issue: Problem with logging in using Safari 12
```
Comment (oddlittlebird) [inline docs/sources/installation/requirements.md]: Why is this here rather than in Getting Started: Logging in to Grafana the first time?

File: docs/sources/installation/windows.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ keywords = ["grafana", "configuration", "documentation", "windows"]
 type = "docs"
 [menu.docs]
 parent = "installation"
-weight = 3
+weight = 400
```
Comment (oddlittlebird) [inline docs/sources/installation/windows.md]: Line 49: Building what? This is a topic about installing, so where should this bit about building Grafana actually be?

File: docs/sources/project/building_from_source.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title = "Building from source"
 type = "docs"
 [menu.docs]
 parent = "installation"
-weight = 5
+weight = 700
```
Comment (oddlittlebird) [inline docs/sources/project/building_from_source.md]: Why is this topic in the Projects folder rather than Installation?

Comment (torkelo) [reply docs/sources/project/building_from_source.md]: because moving it means a URL change and need to add redirect, and we had a plan to restructure all the files and wanted to wait for that and do everything at once to avoid too many 301 redirects (SEO is important for the docs)

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/installation/windows.md]: @torkelo , can you answer this?

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (oddlittlebird) [reply docs/sources/installation/requirements.md]: To clarify: Should I move it?

File: docs/sources/installation/configuration.md [hunk 6]
```diff
@@ -34,6 +34,8 @@ A common problem is forgetting to uncomment a line in the `custom.ini` (or `graf
 > `/etc/grafana/grafana.ini`. This path is specified in the Grafana
 > init.d script using `--config` file parameter.
 
+**Mac:** By default, the Configuration file is located at `/usr/local/etc/grafana/grafana.ini`.
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]:
```suggestion
**macOS:** By default, the configuration file is located at `/usr/local/etc/grafana/grafana.ini`.
```

Maybe we should restructure this section a bit. The default locations vary depending on how they were installed. 

If you downloaded the binaries yourself, the default is `$WORKING_DIR/conf/defaults.ini`.
If you installed using `deb` or `rpm` packages on Linux, the config file is in `/etc/grafana/grafana.ini`.
If you installed using `brew` on macOS, then `/usr/local/etc/grafana/grafana.ini`
If you installed on Windows, then it seems to be same as the binaries.

File: docs/sources/installation/configuration.md [hunk 7]
```diff
@@ -78,9 +80,9 @@ system calls to get the machine name.
 
 ### data
 
-Path to where Grafana stores the sqlite3 database (if used), file based
-sessions (if used), and other data.  This path is usually specified via
-command line in the init.d script or the systemd service file.
+Path to where Grafana stores the sqlite3 database (if used), file based sessions (if used), and other data.  This path is usually specified via command line in the init.d script or the systemd service file.
+
+**Mac:** The default SQLite database is located at `/usr/local/var/lib/grafana`
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]:
```suggestion
**macOS:** The default SQLite database is located at `/usr/local/var/lib/grafana`
```

File: docs/sources/installation/configuration.md [hunk 8]
```diff
@@ -89,14 +91,16 @@ How long temporary images in `data` directory should be kept. Defaults to: `24h`
...
 ### logs
 
-Path to where Grafana will store logs. This path is usually specified via
-command line in the init.d script or the systemd service file.  It can
-be overridden in the configuration file or in the default environment variable
-file.
+Path to where Grafana will store logs. This path is usually specified via command line in the init.d script or the systemd service file.  It can
+be overridden in the configuration file or in the default environment variable file.
+
+**Mac:** By default, the log file should be located at `/usr/local/var/log/grafana/grafana.log`.
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]:
```suggestion
**macOS:** By default, the log file should be located at `/usr/local/var/log/grafana/grafana.log`.
```

File: docs/sources/installation/configuration.md [hunk 9]
```diff
@@ -89,14 +91,16 @@ How long temporary images in `data` directory should be kept. Defaults to: `24h`
 
 ### logs
 
-Path to where Grafana will store logs. This path is usually specified via
-command line in the init.d script or the systemd service file.  It can
-be overridden in the configuration file or in the default environment variable
-file.
+Path to where Grafana will store logs. This path is usually specified via command line in the init.d script or the systemd service file.  It can
+be overridden in the configuration file or in the default environment variable file.
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]: Looks like there's an unintentional line break here?

File: docs/sources/installation/configuration.md [hunk 10]
```diff
@@ -89,14 +91,16 @@ How long temporary images in `data` directory should be kept. Defaults to: `24h`
...
+
+**Mac:** By default, the log file should be located at `/usr/local/var/log/grafana/grafana.log`.
 
 ### plugins
 
-Directory where grafana will automatically scan and look for plugins
+Directory where grafana will automatically scan and look for plugins. Manually or automatically install any plugins here.
+
+**Mac:** By default, the Mac plugin location is: `/usr/local/var/lib/grafana/plugins`.
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]:
```suggestion
**macOS:** By default, the Mac plugin location is: `/usr/local/var/lib/grafana/plugins`.
```

File: docs/sources/installation/configuration.md [hunk 11]
```diff
@@ -89,14 +91,16 @@ How long temporary images in `data` directory should be kept. Defaults to: `24h`
...
-file.
+Path to where Grafana will store logs. This path is usually specified via command line in the init.d script or the systemd service file.  It can
+be overridden in the configuration file or in the default environment variable file.
+
+**Mac:** By default, the log file should be located at `/usr/local/var/log/grafana/grafana.log`.
 
 ### plugins
 
-Directory where grafana will automatically scan and look for plugins
+Directory where grafana will automatically scan and look for plugins. Manually or automatically install any plugins here.
```
Comment (marcusolsson) [inline docs/sources/installation/configuration.md]:
```suggestion
Directory where Grafana will automatically scan and look for plugins. Manually or automatically install any plugins here.
```

File: docs/sources/installation/mac.md [hunk 12]
```diff
@@ -5,12 +5,14 @@ keywords = ["grafana", "configuration", "documentation", "mac", "homebrew", "osx
...
 [menu.docs]
 parent = "installation"
-weight = 4
+weight = 500
 +++
 
 
 # Installing on Mac
 
+This page provides instructions to help you install Grafana on your Mac system. You can either install using homebrew or a binary tar file.
```
Comment (marcusolsson) [inline docs/sources/installation/mac.md]:
```suggestion
This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
```

File: docs/sources/installation/mac.md [hunk 13]
```diff
@@ -5,12 +5,14 @@ keywords = ["grafana", "configuration", "documentation", "mac", "homebrew", "osx
 type = "docs"
 [menu.docs]
 parent = "installation"
-weight = 4
+weight = 500
 +++
 
 
 # Installing on Mac
```
Comment (marcusolsson) [inline docs/sources/installation/mac.md]:
```suggestion
# Installing on macOS
```

File: docs/sources/installation/mac.md [hunk 14]
```diff
@@ -90,11 +75,6 @@ To configure Grafana add a configuration file named `custom.ini` to the
...
-Start Grafana by executing `./bin/grafana-server web`. The `grafana-server`
-binary needs the working directory to be the root install directory (where the
-binary and the `public` folder is located).
-
-## Logging in for the first time
+### Start Grafana
 
-To run Grafana open your browser and go to http://localhost:3000/. 3000 is the default HTTP port that Grafana listens to if you haven't [configured a different port](/installation/configuration/#http-port).
-Then follow the instructions [here](/guides/getting_started/).
+Start Grafana by executing `./bin/grafana-server web`. The `grafana-server` binary needs the working directory to be the root install directory (where the binary and the `public` folder is located).
```
Comment (marcusolsson) [inline docs/sources/installation/mac.md]:
```suggestion
Start Grafana by executing `./bin/grafana-server`. The `grafana-server` binary needs the working directory to be the root install directory (where the binary and the `public` folder is located).
```

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (marcusolsson) [reply docs/sources/installation/requirements.md]: I think it's more related to the section on supported platforms, rather than Getting started. As in, these are the platforms we support, but we're also seeing this issue.

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Nice! Added a few suggestions, mainly on being consistent with macOS naming.

File: docs/sources/installation/configuration.md (hunk 6 above)
Comment (marefr) [reply docs/sources/installation/configuration.md]: Sounds good. If we don't want to have this on multiple places, maybe we can update each installation page and link to them? For example linking to https://grafana.com/docs/installation/rpm/#package-details for rpm.

File: docs/sources/installation/configuration.md [hunk 15]
```diff
@@ -78,9 +80,9 @@ system calls to get the machine name.
 
 ### data
 
-Path to where Grafana stores the sqlite3 database (if used), file based
-sessions (if used), and other data.  This path is usually specified via
-command line in the init.d script or the systemd service file.
+Path to where Grafana stores the sqlite3 database (if used), file based sessions (if used), and other data.  This path is usually specified via command line in the init.d script or the systemd service file.
```
Comment (marefr) [inline docs/sources/installation/configuration.md]: Sessions is no longer in use

File: docs/sources/installation/requirements.md [hunk 16]
```diff
@@ -7,23 +7,23 @@ type = "docs"
...
-### Supported
+The following operating systems are supported for Grafana installation:
 
 - [Debian / Ubuntu](/installation/debian)
 - [RPM-based Linux (CentOS, Fedora, OpenSuse, RedHat)](/installation/rpm)
 - [Mac OS X](/installation/mac)
 - [Windows](/installation/windows)
 
-### Unsupported
+## Unsupported
```
Comment (marefr) [inline docs/sources/installation/requirements.md]: This needs to be `Unsupported operating systems` since you've moved this to be a H2 instead of H3 - earlier the unsupported was a sub-header of `Operating systems`.

**Review by marefr — DISMISSED** [review]
Good job. Added some comments/request for changes.

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (oddlittlebird) [reply docs/sources/installation/requirements.md]: @ivanahuckova , can you clarify where this should go?

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/installation/windows.md]: @ivanahuckova , can you answer this?

File: docs/sources/installation/configuration.md (hunk 6 above)
Comment (oddlittlebird) [reply docs/sources/installation/configuration.md]: In the longer term, I am hoping to consolidate and simplify the Installation and Getting Started documentation, probably by having merging the sections and organizing them by system:
* Getting started with Windows
* Getting Started with MacOS
* And so on

ALL of this needs to be rewritten. This first PR is just trying to begin sorting content into the right places, and I am starting with the obvious, easy stuff. For example, configuration information should not be in the installation docs. Once everything is sorted into its proper "bucket," then we can start making decisions about how else to organize it.

File: docs/sources/installation/configuration.md (hunk 15 above)
Comment (oddlittlebird) [reply docs/sources/installation/configuration.md]: So, what should I do with this content? 

Also, should "systemd" be "system.d"?

File: docs/sources/installation/requirements.md (hunk 16 above)
Comment (oddlittlebird) [reply docs/sources/installation/requirements.md]: Excellent, thanks for clarifying that!

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (ivanahuckova) [reply docs/sources/installation/windows.md]: My opinion - I think we shouldn't mix installation documentation with building from source documentation - and we should keep those 2 separate. Therefore I would remove this information from here. Also, I have found documentation related directly to [building from source](https://grafana.com/docs/project/building_from_source/), where is this exact information mentioned as well. However, I couldn't find this file in this repo - I think it makes sense to have it here and move it from the other location. 
What do you think? @torkelo @marcusolsson

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (ivanahuckova) [reply docs/sources/installation/requirements.md]: I was browsing the docs and honestly, I don't know. Maybe we could have a separate documentation for known issues? Or we can just add Note 4 with this information. Safari13 has been already released, so we can link the issue with Safari 12 and add a note to upgrade to Safari 13 or to Grafana 6.3.3.
image

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (marcusolsson) [reply docs/sources/installation/windows.md]: I think we should differentiate between people who just want to build the latest version of Grafana, and people who are looking to develop Grafana. The latter should be covered in the [Developer guide](https://github.com/grafana/grafana/blob/master/contribute/developer-guide.md), but I think we should also consider having a stripped down guide for just building the binary. 

At this point though, everything in [Building from source](https://grafana.com/docs/project/building_from_source/) is covered in the Developer guide, so maybe we should replace it with a reference to the Developer guide for now?

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (oddlittlebird) [reply docs/sources/installation/requirements.md]: Thanks for pointing that out, @ivanahuckova. I think the Safari issue is probably covered under Note 1, so we can remove the specific bit about Safari 12. I will also clarify that we recommend users should upgrade their browsers to the latest version.

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/installation/windows.md]: I like that plan, @marcusolsson. We don't want to confuse the laypeople, and it matches the way we've been organizing the contribution docs in the repo. 

Regarding the specific paragraph in windows.md, is that captured somewhere else already as well? Is it safe to delete it from windows.md?

File: docs/sources/installation/requirements.md (hunk 3 above)
Comment (ivanahuckova) [reply docs/sources/installation/requirements.md]: @oddlittlebird yes, that's true! Great point.

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (marcusolsson) [reply docs/sources/installation/windows.md]: I don't think it's mentioned elsewhere right now. Either we leave that part as is, or we move it to the Developer guide?

File: docs/sources/installation/configuration.md (hunk 15 above)
Comment (marcusolsson) [reply docs/sources/installation/configuration.md]: [systemd](https://github.com/systemd/systemd)

File: docs/sources/installation/windows.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/installation/windows.md]: Let's leave it here for now.

File: docs/sources/installation/configuration.md (hunk 15 above)
Comment (oddlittlebird) [reply docs/sources/installation/configuration.md]: Sounds good. I saw `init.d` and wondered if the systemd should be the same format.

**Comment by oddlittlebird** [comment]
I deleted the content from Building From Source and added a link to the Developer Guide in that file.

I see that there are still outstanding change requests from @marcusolsson  and @marefr, but everything is showing as either resolved or outdated in my view.

Is there anything else that needs to be done right now for this PR? I will be overhauling the Installation and Getting Started sections completely in a couple weeks.

**Review by marcusolsson — APPROVED** [review]
LGTM!
